project/build.properties

```diff
@@ -1 +1 @@
-sbt.version = 0.13.13
\ No newline at end of file
+sbt.version = 1.2.8
```

Comment:
We don't rely on anything fancy here, is 1.3 feasible then?

Reply:
let me try..

Reply:
with 1.3.13:
```
# Unit tests step
[info] Compiling 143 Scala sources to /home/circleci/project/target/scala-2.12/test-classes ...
/usr/share/sbt/bin/sbt-launch-lib.bash: line 58:   789 Killed                  "$@"
```
^ i tried to increase memory but it doesn't worked, so i decided to turn back to 1.2.8

Reply:
👍 
Did you update launcher script as well? It has changed a bit between 1.2.x and 1.3 to accomodate for specifically different approach to memory